src/components/Reviews.jsx

```diff
@@ -5,6 +5,10 @@ import ReviewCard from "./ReviewCard";
 const Reviews = () => {
   const [currReviews, setCurrReviews] = useState([]);
 
+  if (currReviews.length === 0) {
```

Comment:
We would need to create a state for this. This method does not work. Let us know if you need some help with this. Here is a section of the notes to do with this:
https://notes.northcoders.com/courses/js-front-end/useEffect#loading-patterns